### ISSUE TITLE
[Windows] "crypto/x509: system root pool is not available on Windows"

### ISSUE THREAD

**Issue by placaze** [deviceinsight/kafkactl#108]
With Windows, we see the warning `error reading system cert pool: crypto/x509: system root pool is not available on Windows` everytime *kafkactl* reaches the brokers.

Example:
```
MINGW64 ~/Downloads
$ ./kafkactl.exe get topics --config-file kafkactl-dev.yaml
error reading system cert pool: crypto/x509: system root pool is not available on Windows
error reading system cert pool: crypto/x509: system root pool is not available on Windows
TOPIC                                                           PARTITIONS
[...]
```

I saw a workaround here: https://github.com/golang/go/issues/16736#issuecomment-540373689. I don't know if it can help.

**Comment by random-dwi** [comment]
Hey @placaze 
Thank you for reporting. The solution is probably just to avoid fetching systemCerts for Windows somewhere here:
https://github.com/deviceinsight/kafkactl/blob/master/internal/common-operation.go#L217

**Comment by d-rk** [comment]
@placaze this will be fixed with https://github.com/golang/go/commit/3544082f75fd3d2df7af237ed9aef3ddd499ab9c and will probably be available with go 1.18.
So we will just wait for this.
